resources/persist/src/lib.rs

```diff
@@ -111,13 +152,53 @@ mod tests {
         assert_eq!(result, "test");
     }
 
+    #[test]
+    fn test_list() {
+        let persist = PersistInstance {
+            service_name: ServiceName::from_str("test_list").unwrap(),
+        };
+
+        persist.save("test_list", "test_list").unwrap();
+
+        let result = vec!["shuttle_persist/test_list/test_list".to_string()];
```

Comment:
Should the resulting list item really contain the full path like this?

Reply:
Fixed, list returns the key name only.